src/cargo/sources/git/utils.rs

```diff
@@ -732,6 +745,24 @@ pub fn fetch(
     })
 }
 
+fn fetch_with_cli(
+    repo: &mut git2::Repository,
+    url: &Url,
+    refspec: &str,
+    config: &Config,
+) -> CargoResult<()> {
+    let mut cmd = process("git");
+    cmd.arg("fetch")
+        .arg("--tags") // fetch all tags
+        .arg("--quiet")
```

Comment:
is this for feature parity with `libgit2`'s variant?

Reply:
Indeed yeah!